addons/project/data/project_demo.xml

```diff
@@ -251,7 +602,44 @@
             <field name="user_id" eval="False"/>
             <field name="project_id" ref="project.project_project_2"/>
             <field name="name">Unit Testing</field>
-            <field name="stage_id" ref="project_stage_0"/>
+            <field name="stage_id" ref="project_stage_2"/>
+            <field name="create_date" eval="DateTime.now() - relativedelta(months=5)"/>
+        </record>
+        <record id="project_task_26_mail_message_1" model="mail.message">
+            <field name="model">project.task</field>
+            <field name="res_id" ref="project.project_task_26"/>
+            <field name="message_type">notification</field>
+            <field name="subtype_id" ref="mt_task_stage"/>
+            <field name="date" eval="DateTime.now() - relativedelta(months=3)"/>
```

Comment:
I see `project_task_26_mail_message_1` and `project_task_26_mail_message_2` have the same date perhaps change one of both ? 🤔 
```suggestion
            <field name="date" eval="DateTime.now() - relativedelta(months=4)"/>
```

Reply:
@xavierbol , Done! Thanks